### DISCUSSION TITLE
Eleccion de Proyecto

### DISCUSSION THREAD

**Discussion by carlos319** [rramosp/ai4eng.v1#277] [00 - General]
Un saludo para tod@s, en nuestro grupo hemos encontrados posibles datasets, pero queremos aclarar algunas dudas al respecto y nos gustaria que si es posible @Felipe-RA nos acompañes 15 a 30min en una seccion por meet mañana 19-de Sept-2023, gracias!

(Sin título)
Lunes, 18 de septiembre · 10:30 – 11:30pm
Zona horaria: America/Bogota
Información para unirse con Google Meet
Enlace de la videollamada: https://meet.google.com/brw-ejyj-vpo

**Comment by Felipe-RA** [comment]
Vale, no  hay problema.

Mañana 10.30 pm?

**Reply by carlos319** [reply]
perdon 4pm puede? a esa hora de pronto por jornada laboral no nos da

**Reply by Felipe-RA** [reply]
Hola carlos, era una pregunta pues pusiste  10.30 a 11.30 en tu post.

Entiendo la limitación, lamentablemente esta semana para asesorias personales solo tengo posibilidad en horarios algo extremos, 9.00 a 9.30 p.m es lo más temprano que te puedo colaborar.

Recuerda que también tienes la opción de pasarte por las sesiones sincronicas 10-12 m-j y lo puedes ver.

Estoy al tanto

**Reply by carlos319** [reply]
dale, le comunicaré a mi grupo la hora disponible suya, gracias te estare confirmando lo antes posible por este medio. gracias y feliz noche!

**Reply by carlos319** [reply]
Buenas Tardes @Felipe-RA, le comfirmamos la asistencia hoy de 9 a 9:30pm. gracias!"

**Reply by carlos319** [reply]
Buenas noches @Felipe-RA, por fa nos separas un espacio para mañana 9pm. gracias
Enlace de la videollamada: https://meet.google.com/brw-ejyj-vpo

**Reply by Felipe-RA** [reply]
Hola @carlos319 

No habia visto, pero si. Adelante nos vemos a las 9

EDIT: 😕
